tools/system_libs.py

```diff
@@ -41,7 +41,7 @@ def read_symbols(path, exclude=None):
       symbols = filter(lambda symbol: symbol not in exclude, symbols)
     return set(symbols)
 
-  default_opts = []
+  default_opts = ['-Werror']
```

Comment:
This code is also used by musl on x86/x64, where long double is 80 bit and also not the same as double, so if it's a bug for wasm it's a bug for them too. Reporting it upstream and fixing it locally makes sense.